asset-transfer-basic/rest-api-typescript/scripts/generateEnv.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+${AS_LOCAL_HOST:=true}
+
+: "${TEST_NETWORK_HOME:=../..}"
+: "${CONNECTION_PROFILE_FILE_ORG1:=${TEST_NETWORK_HOME}/organizations/peerOrganizations/org1.example.com/connection-org1.json}"
+: "${CERTIFICATE_FILE_ORG1:=${TEST_NETWORK_HOME}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem}"
```

Comment:
I'll have a try

Reply:
The User1 credentials work, so I've switched those over hyperledgendary/fabric-rest-sample#53
I'll leave the variable names as they are since they are to configure any suitable org1 user, not necessarily User1